packages/core/content-manager/server/tests/relations.test.e2e.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
+
+const { createTestBuilder } = require('../../../../../test/helpers/builder');
+const { createStrapiInstance } = require('../../../../../test/helpers/strapi');
+const { createAuthRequest } = require('../../../../../test/helpers/request');
+
+let strapi;
+let rq;
+const data = {
+  products: [],
+  shops: [],
+};
+
+const compo = {
+  displayName: 'compo',
+  attributes: {
+    name: {
+      type: 'string',
+    },
+    compoProducts: {
+      type: 'relation',
+      relation: 'manyToMany',
+      target: 'api::product.product',
+    },
+  },
+};
+
+const productModel = {
+  attributes: {
+    name: {
+      type: 'string',
+    },
+  },
+  displayName: 'Product',
+  singularName: 'product',
+  pluralName: 'products',
+  description: '',
+  collectionName: '',
+};
+
+const shopModel = {
+  attributes: {
+    name: {
+      type: 'string',
+    },
+    products: {
+      type: 'relation',
+      relation: 'manyToMany',
+      target: 'api::product.product',
+      targetAttribute: 'shops',
+      inversedBy: 'shops',
+    },
+    myCompo: {
+      type: 'component',
+      repeatable: false,
+      component: 'default.compo',
+    },
+  },
+  displayName: 'Shop',
+  singularName: 'shop',
+  pluralName: 'shops',
+};
+
+describe('Relations', () => {
+  const builder = createTestBuilder();
+
+  beforeAll(async () => {
+    await builder
+      .addContentTypes([productModel])
+      .addComponent(compo)
+      .addContentTypes([shopModel])
+      .build();
+
+    strapi = await createStrapiInstance();
+    rq = await createAuthRequest({ strapi });
+
+    const { body: createdProduct1 } = await rq({
+      method: 'POST',
+      url: '/content-manager/collection-types/api::product.product',
+      body: { name: 'Skate' },
+    });
+    const { body: createdProduct2 } = await rq({
+      method: 'POST',
+      url: '/content-manager/collection-types/api::product.product',
+      body: { name: 'Candle' },
+    });
```

Comment:
minor thing. Maybe because I am still learning all the routes, but do you think it would helpful doing something like this?

```suggestion
    const createdProduct1 = await createEntry('api::product.product', {
      name: 'Skate',
    });

    const createdProduct2 = await createEntry('api::product.product', {
      name: 'Candle',
    });
```

Maybe you could have one for getting an entry and creating one.

```js

async function createEntry(uid, data) {
  const { body } = await rq({
    method: 'POST',
    url: `/content-manager/collection-types/${uid}`,
    body: data,
  });
  return body;
}

async function getEntry(uid, qs = undefined) {
  return rq({
    method: 'GET',
    url: `/content-manager/collection-types/${uid}`,
    qs
  });
}
```

Reply:
Yes why not, it's seems reasonable to me